lib/jxl/dec_16bit_reconstruct.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#include "lib/jxl/dec_cache.h"
+
+namespace jxl {
+void FinalizeImageRect16(Image3F* input_image, const Rect& input_rect,
```

Comment:
document meeeee plz. For example, explain why is this unreachable in other than NEON? when should this be used?